Cargo.toml

```diff
@@ -1,7 +1,21 @@
 [package]
 name = "aurebuildcheck-rs"
-version = "0.1.0"
-authors = ["Matthias Krüger <matthias.krueger@famsik.de>"]
+version = "0.2.0"
+authors = [
+    "Matthias Krüger <matthias.krueger@famsik.de>",
+    "Marc Mettke <marc@itmettke.de>"
+]
+repository = "https://github.com/matthiaskrgr/aurebuildcheck-rs"
+homepage = "https://github.com/matthiaskrgr/aurebuildcheck-rs"
+license = "MIT"
+readme = "README.md"
+description = "Archlinux package checker to identify packages which may need a rebuild"
 
 [dependencies]
-rayon = "0.9" # parallelize
+rayon = "0.9.0"
+clap = "2.29.4"
+json = "0.11.13"
+
+[profile.release]
+lto = true
+panic = 'abort'
```

Comment:
I would like to keep ````codegen-units=1````
It will slow down building in release mode a bit but it will speed up linking a lot and will allow for better optimizations.
https://github.com/rust-lang/rust/issues/48025

Reply:
I saw your commit for rpm - will do

Reply:
Ah yeah. The machine I made the rust prototype on runs fedora so I just used rpm to be able to do some quick "does it run" tests. Looks like I forgot to merge the arch branch, but whatever.